### PR TITLE
Added Experiences Section

### PR DESCRIPTION
**Pull request by HenryHengZJ**
I have added the Experience Section.
The Experience section can be enabled or disabled via setting viewExperiences in portfolio.js.

### CODE REVIEW

**Comment by naveen521kk** [comment]
It would good if font size is reduced on the screen size reduces. Further, doesn't look good on mobiles. Make it match other components.

File: src/containers/workExperiences/workExperiences.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./workExperiences.css";
+import ExperienceCard from "../../components/experienceCard/ExperienceCard";
+import { workExperiences } from "../../portfolio";
+import { Fade } from "react-reveal";
+
+
+export default function Experiences() {
```
Comment (kartik918) [inline src/containers/workExperiences/workExperiences.js:8]: Let's keep the export name consistent to the file name for standards and less confusion.

Also rename file to `workExperience` and update wherever required in other files too.

File: src/portfolio.js [hunk 2]
```diff
@@ -115,6 +115,36 @@ const techStack = {
...
+      companylogo: "https://www.referralcandy.com/wp-content/uploads/2017/06/quora-featured-image-2-740x740.png",
+      date: "May 2017 – May 2018",
+      descrip: "Write clean, maintainable code using best industry standards in Web languages, and integrate with existing APIs."
+    },
+    {
+      role: "Software Engineer Intern",  
+      company: "Airbnb",
+      companylogo: require("./assests/images/airbnbLogo.jpg"),
+      date: "Jan 2015 – Sep 2015",
+      descrip: "Develop and maintain new/current software applications, meeting user requirements and complying with technical specifications."
```
Comment (kartik918) [inline src/portfolio.js:143]: Let's change `descrip` prop name to `desc` for consistency and readability. Also update accordingly in other files.
Switch `desc` to Lorem Ipsum

**Comment by HenryHengZJ** [comment]
> It would be more appropriate to use placeholder information instead of accurate dates and details (something like Lorem Ipsum, we can stick to company icons or some other placeholder images to show the color extraction feature).
> 
> Also, it seems to be slightly inconsistent on smaller screens.
> 
> image
> 
> Love the card design BTW!

Thanks for the changes suggestions. I noticed the inconsistent look at iPhone5/SE, the same goes for Achievement and Certifications. Changes needed to be made there too.

image

**Comment by kartik918** [comment]
Great find!

We should investigate what's causing this in the first place before moving to a fix.

**Comment by HenryHengZJ** [comment]
The problem is at the CSS file of the grid divider:
`grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));`
If you reduce 300px to 290px, it fixes the problem.

**Comment by kartik918** [comment]
Even for larger screens? I don't want it to look slightly off from the centre. Thanks!

**Comment by HenryHengZJ** [comment]
The problem only occurs when tested with Iphone5/SE, works fine on larger screens. Making it 290px will not shift the grid, the only thing changed is the min card's width is now 290px when screen size reduces. 
image

Below is the image when tested with Iphone5/SE. 

image

**Comment by kartik918** [comment]
Having slightly smaller width isn't a "problem" as severe as having cards offset from the center. I think this looks great. Push the requested changes and we are good to go!

File: src/portfolio.js (hunk 2 above)
Comment (HenryHengZJ) [reply src/portfolio.js:143]: do you want to change the date to Lorem Ipum as well?

Comment (kartik918) [reply src/portfolio.js:143]: Nah, just add some place holder date or let it be as it is right now.

File: README.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ If you created something awesome and want to contribute then feel free to open P
 ## Sections 
 ✔️ Summary and About me\
 ✔️ Skills \
+✔️ Experiences \
```
Comment (kartik918) [inline README.md:24]: Let's change this to "Work Experience", seems more appropriate.

Comment (kartik918) [reply README.md]: Nevermind, did it myself.

**Comment by kartik918** [comment]
@saadpasta you can add your own work experience in a follow up PR or we can move to neutral filler information for the demo build. Whatever you decide on this, I'm for it.

**Comment by naveen521kk** [comment]
You @kartik918 forgot to add @HenryHengZJ as a contributor.

**Comment by kartik918** [comment]
@allcontributors please add @HenryHengZJ for code and design

**Comment by HenryHengZJ** [comment]
Thanks guys for the help and suggestions!

**Pull request merged**